### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and settings_Changing the User settings_ Screen reader: Voiceover is not announcing the message which appears when focus comes on "Auto".

### ISSUE THREAD

**Issue by kjawale99** [microsoft/vscode#164699]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; #Voiceover; #Screenreader; #Info and Relationships;

### Environment and OS details:
Visual Studio Code
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor 
2. Open the command Pallet and type "setting".
3. Using down-arrow key select user setting.
4. Type the word "Auto " and select one of the suggestion given by the vs code editor.
5. Type the "Tab size" and select the option from the auto suggestion.
6. Verify that Voiceover is not announcing the message which appears when focus comes on "Auto".

### Actual Result:
Voiceover is not announcing the message (Remove adjacent quotes ....) which appears when focus comes on "Auto".

### Expected Result:
Voiceover should announce the message (Remove adjacent quotes ....) which appears when focus comes on "Auto".

### Attachment:
![message](https://user-images.githubusercontent.com/98745784/197955500-494e93b0-438e-4e22-9366-8c232174be82.gif)

**Comment by isidorn** [comment]
VoiceOver nicely reads for me whenever a settings value changes in the dropdown.
Can you upload a gif in good quality so I can actually see what you are doing? Thanks

**Comment by kjawale99** [comment]
@isidorn I have attached a gif in good quality

**Comment by isidorn** [comment]
@kjawale99 thanks. But I still do not understand what the issue is. In the gif I see that VoiceOver nicely reads the selected value? The only problem I see is that VoiceOver appends "default" to the word "auto" but that is a VoiceOver issue they are aware of.

**Comment by kjawale99** [comment]
@isidorn when the focus comes on Auto a message is displayed below (Remote adjacent...). This message is not read by voiceover.

**Comment by isidorn** [comment]
@kjawale99 aha good catch, thanks!

Forwarding to @rzhao271 
Gist - description of setting values is not being read by Screen Readers. We should somehow add it to the aria label of the dropdown list items

**Comment by rzhao271** [comment]
Confirmed on macOS with Voiceover.
NVDA is fine

**Comment by isidorn** [comment]
Thanks for trying this out @rzhao271.
Marking it as upstream since it is a most likely a voice over bug. I have went ahead and sent an email to our VoiceOver friends.

**Comment by isidorn** [comment]
This is an upstream issue thus I am going to go ahead and close this as per the a11y process - so our grade does not depend on a VoiceOver bug. 
It is up to VoiceOver to fix this.

**Comment by rzhao271** [comment]
Yesterday, I also tried using `aria-describedby` just in case that could help VoiceOver pick up the right text to read, but it couldn't.

I also noticed that VoiceOver wasn't reading the `option` elements, but the actual elements displayed by the dropdown. Setting an aria-label on the `option` elements therefore didn't work, but setting an aria-label on the displayed elements didn't work, either.
